modules/extensions-support/src/main/java/org/mule/module/extension/internal/introspection/AnnotationsBasedDescriber.java

```diff
@@ -343,6 +354,25 @@ private void declareOperationParameters(Method method, OperationDescriptor opera
             {
                 operation.withModelProperty(ConnectionTypeModelProperty.KEY, new ConnectionTypeModelProperty(parsedParameter.getType().getRawType()));
             }
+
+            UseConfig useConfig = parsedParameter.getAnnotation(UseConfig.class);
+            if (useConfig != null)
+            {
+                operation.withModelProperty(UseConfigTypeModelProperty.KEY, new UseConfigTypeModelProperty(parsedParameter.getType().getRawType()));
+            }
+        }
+    }
+
+    private void checkAnnotationIsNotUsedMoreThanOnce(Method method, Class annotationClass)
+    {
+        Stream<java.lang.reflect.Parameter> parametersStream = Arrays
+                .asList(method.getParameters())
+                .stream()
+                .filter(parameter -> parameter.isAnnotationPresent(annotationClass));
+
+        if (parametersStream.collect(Collectors.toList()).size() > 1)
+        {
+            throw new IllegalModelDefinitionException(String.format("The @%s annotation can not be used more than a time in a @%s method. Affected method [%s]", annotationClass.getSimpleName(), Operation.class.getSimpleName(), method.getName()));
```

Comment:
rephrase as "Method X defined in Class Y of extension Z uses the annotation %s more than once